client/src/layout/corporateDesign.css

```diff
@@ -0,0 +1,12 @@
+/* changes to the variables here also have to be made in corporateDesign.js !!! */
```

Comment:
Please avoid the usage of three exclamation marks (!!!)

Reply:
fair haha changed it to one exclamation mark